src/turnkeyml/build/export.py

```diff
@@ -279,9 +280,31 @@ def fire(self, state: build.State):
         default_warnings = warnings.showwarning
         warnings.showwarning = _warn_to_stdout
 
+        stats = fs.Stats(
+            state.cache_dir, state.config.build_name, state.evaluation_id
+        )
+
+        # Verify if the exported model matches the input torch model
+        try:
+            # The `torch.onnx.verification.find_mismatch()` takes input arguments to the
+            # model as `input_args (Tuple[Any, ...])`
+            export_verification = torch.onnx.verification.find_mismatch(
+                state.model,
+                tuple(state.inputs.values()),
+                opset_version=state.config.onnx_opset)
+            is_export_valid = not export_verification.has_mismatch()
```

Comment:
Suggest always using one data type for `is_export_valid`. Right now it can hold a `bool` or a `str` depending on the code path.

Reply:
Changed to use `str` for all cases

Reply:
nit: please make them all lowercase, since thats what we do with all the other values